k8s/base/gcp_gke/kustomization.yaml

```diff
@@ -5,6 +5,7 @@ resources:
   - configmaps.yaml
   - redis.yaml
   - deployments.yaml
+  - mpm-storage-pvc.yaml
```

Comment:
Can we rename the file to skip the `mpm-` prefix here? Just to be consistent with other files in this folder.